From 2f703241720c44963ebb95e9c2ad82d7dc86eadc Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Fri, 16 Apr 2021 06:44:21 +0000
Subject: [PATCH] libstdc++: do not declare a set of const types

GNU libstdc++ std::set explicitely forbids creating a std::set
of const types.
---
 chrome/browser/browsing_data/third_party_data_remover.cc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/chrome/browser/browsing_data/third_party_data_remover.cc b/chrome/browser/browsing_data/third_party_data_remover.cc
index 20271b5..ef3cfeb 100644
--- a/chrome/browser/browsing_data/third_party_data_remover.cc
+++ b/chrome/browser/browsing_data/third_party_data_remover.cc
@@ -14,7 +14,7 @@
 namespace {
 
 bool DoesOriginHaveThirdPartyAccessRecord(
-    const std::set<const url::Origin>& third_party_storage_origins,
+    const std::set<url::Origin>& third_party_storage_origins,
     const url::Origin& origin,
     storage::SpecialStoragePolicy* policy) {
   return third_party_storage_origins.find(origin) !=
@@ -25,7 +25,7 @@ void OnGetStorageAccessRecords(
     base::OnceClosure closure,
     content::BrowserContext* context,
     std::vector<AccessContextAuditDatabase::AccessRecord> records) {
-  std::set<const url::Origin> origins;
+  std::set<url::Origin> origins;
   for (const auto& record : records) {
     origins.insert(std::move(record.origin));
   }
-- 
2.26.3

